test/JDBC/input/views/sys-objects-vu-prepare.sql

```diff
@@ -0,0 +1,36 @@
+CREATE DATABASE sys_objects_vu_prepare_db1;
```

Comment:
do not use a separate database unless you specifically want to test multi-db/cross-db

Reply:
Thanks for your comment. The purpose of creating a db here is to make sure any procedure objects created in this DB are not present when we switch context to `master` db. This is the [line of code that tests this](https://github.com/babelfish-for-postgresql/babelfish_extensions/pull/452/files/e5c95054ea297101cce8d7c4633b83a846637b61#diff-b9940fec8e94414b06006a63a564d1d16d56f62f476d047255fe54fd25d96056R39). Is this still a valid case of cross-db testing?

Reply:
this is fine as we are creating database for specific testing purpose.